example.env

```diff
@@ -1 +0,0 @@
-REACT_APP_APIKEY=
```

Comment:
We need to restore this

Reply:
Okay, I'll make the changes. Are these the things that are causing tests to fail?

Reply:
No, that's the code style (prettier). We'll need a `yarn format` run to fix that

Reply:
Can you take a look now? :D